src/facility.ts

```diff
@@ -0,0 +1,72 @@
+import axios from "axios";
+
+var axiosInstance = axios.create({
+    baseURL: process.env.REACT_APP_FACILITY_API_PREFIX
```

Comment:
I don't see where you define `REACT_APP_FACILITY_API_PREFIX`

Reply:
fixed in .env file - https://github.com/bluesky/bluesky-webclient/pull/21/commits/2657b6de1dca8bb054ef5facddf5660846de506f